electron/src/features/entries/EntryEditor.tsx

```diff
@@ -284,9 +345,22 @@ function EntryEditor() {
   } else {
     if (!entry?.transcriptions[0]) {
       content = (
-        <Card shadow="md" p="lg">
-          <Text>No Transcription Found</Text>
-        </Card>
+        <Center>
+          <Stack>
+            <Title order={3}>No Transcription Found</Title>
+            <Button
+              onClick={() => {
+                //@ts-expect-error entry is always going to be defined here
```

Comment:
This should be handled with a check for entry rather than removing the typing

Reply:
Oversight on my part. Thought I'd already changed that 😅

Just pushed a fix

Reply:
Added an else clause as well